### ISSUE TITLE
RecyclerView Adapter Issue 

### ISSUE THREAD

**Issue by vsahu1986** [eowise/recyclerview-stickyheaders#47]
When we update , add ,delete an item from Adapter , or Attach a new Adapter to RecyclerView when in Sticky mode , lots of White Space appear between sticky header and Recycler View 

it is just because of Translation Y

**Comment by vipinhelloindia** [comment]
+1

**Comment by TheReprator** [comment]
yes, when i add new item to recyclerview, it jumps over to sticky header

**Comment by davideas** [comment]
About adding, removing, moving, you can check my answer at this issue (A new way to display Sticky Headers) #52
